saleor/graphql/order/filters.py

```diff
@@ -26,6 +28,13 @@ def get_payment_id_from_query(value):
         return None
 
 
+def get_order_id_from_query(value):
+    if value.startswith("#"):
```

Comment:
little comment why this can start with `#`?

Reply:
In the dashboard, we display Order number with # as the prefix.
![image](https://user-images.githubusercontent.com/43955230/117292476-88d28400-ae70-11eb-977c-4e2697bd98d5.png)